### ISSUE TITLE
Atropos error in ChIP-SEq: Workers are still alive and haven't returned summaries

### ISSUE THREAD

**Issue by mchimenti** [bcbio/bcbio-nextgen#2643]
Hello:
I'm trying to run ChIP-Seq analysis using the standard ChIP-Seq pipeline. The pipeline runs through to completion, but the trimming step takes a very long time and the log fills up with these errors: 

[2019-01-22T23:48Z] 2019-01-22 17:48:34,698 INFO: This is Atropos 1.1.16 with Python 3.6.4
[2019-01-22T23:48Z] 2019-01-22 17:48:34,712 INFO: Trimming 4 adapters with at most 10.0% errors in paired-end mode ...
[2019-01-22T23:48Z] 2019-01-22 17:48:34,755 INFO: Starting 15 worker processes
[2019-01-22T23:50Z] 2019-01-22 17:50:20,659 INFO: Starting 1 worker processes
[2019-01-22T23:51Z] 2019-01-22 17:51:21,094 ERROR: Waiting on worker summaries for 60.3 seconds
[2019-01-22T23:51Z] 2019-01-22 17:51:21,094 ERROR: Workers are still alive and haven't returned summaries: 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15
[2019-01-22T23:51Z] 2019-01-22 17:51:26,120 ERROR: Waiting on worker summaries for 65.3 seconds
[2019-01-22T23:51Z] 2019-01-22 17:51:26,120 ERROR: Workers are still alive and haven't returned summaries: 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15
[2019-01-22T23:51Z] 2019-01-22 17:51:31,145 ERROR: Waiting on worker summaries for 70.4 seconds

****
This goes on for several hundred seconds and then I think the pipeline moves onto the next sample.  
Any insight into what is going on would be appreciated.  I couldn't find this issue specifically at the "atropos" github or here either. 

Thank you.  

*****

Config: 

details:
- analysis: chip-seq
  metadata:
     batch: group_1
     phenotype: chip # or input
  genome_build: GRCh37
  algorithm:
    aligner: bwa
    adapters: [truseq]
    trim_reads: read_through
    peakcaller: [macs2]
  resources:
    macs2:
      options: ["-q 0.05"]

*********
Running bcbio 1.0.8 on a distributed Linux cluster with SGE with:

#!/bin/bash
#$ -pe smp 28
#$ -S /bin/bash
#$ -cwd
#$ -M michael-chimenti@uiowa.edu
#$ -m aes
#$ -j y
# pass the ./work directory of the bcbio project as the first argument
#pass the yaml file as the second argument

module use /Dedicated/IIHG-argon/modulefiles
module load bcbio/1.0.8
bcbio_nextgen.py $1 -t local -n 28

*****

**Comment by ohofmann** [comment]
I am seeing those as well, usually for large files that take a long time to write back to disk - but it never seems to cause issues here, i.e., the files eventually end up in the place where they are supposed to be and the workflow moves on.

Do samples get skipped in your case?

**Comment by mchimenti** [comment]
Hi ohofmann,

No, samples do not appear to be skipped.  At the end, the BAM alignments are all written out along with the MACS2 peak lists.   As well, the "trimmed" directory in "work" does contain all of the trimmed fastq input files.  

So I guess this is a "non-issue" but it looked problematic from the logs.  Would be nice to have some reassurance that nothing bad is happening at this step.

**Comment by chapmanb** [comment]
Michael and Oliver;
Thanks for the report and discussion. Those messages aren't anything to worry about, just a side effect of waiting during multicore trimming:

https://github.com/jdidion/atropos/blob/1.1/doc/guide.rst#multi-threading

Let us know if you run into any issues and happy to dig more. Thanks again.
